bitnami/kiam/README.md

```diff
@@ -0,0 +1,375 @@
+| `server.nodeSelector`                       | Node labels for pod assignment                                                              | `{}` (evaluated as a template)           |
+| `server.tolerations`                        | Tolerations for pod assignment                                                              | `[]` (evaluated as a template)           |
+| `server.podLabels`                          | Extra labels for kiam pods                                                                  | `{}`                                     |
+| `server.podAnnotations`                     | Annotations for kiam pods                                                                   | `{}`                                     |
+| `server.priorityClassName`                  | Server priorityClassName                                                                    | `nil`                                    |
+| `server.lifecycleHooks`                     | LifecycleHooks to set additional configuration at startup.                                  | `{}` (evaluated as a template)           |
+| `server.extraVolumeMounts`                  | Optionally specify extra list of additional volumeMounts for kiam container(s)              | `[]`                                     |
+| `server.extraVolumes`                       | Optionally specify extra list of additional volumes for kiam pods                           | `[]`                                     |
+| `server.initContainers`                     | Add additional init containers to the kiam pods                                             | `{}` (evaluated as a template)           |
+| `server.sidecars`                           | Add additional sidecar containers to the kiam pods                                          | `{}` (evaluated as a template)           |
+
+### kiam agent parameters
+
+| Parameter                                   | Description                                                                                | Default                                  |
+|---------------------------------------------|--------------------------------------------------------------------------------------------|------------------------------------------|
+| `agent.enabled`                             | Deploy the kiam agent                                                                      | `true`                                   |
+| `agent.containerPort`                       | HTTPS port to expose at container level                                                    | `8443`                                   |
+| `agent.allowRouteRegexp`                    | Regexp with the allowed paths for agents to redirect                                       | `nil`                                    |
+| `agent.iptables`                            | Have the agent modify the host iptables rules                                              | `false`                                  |
```

Comment:
This option name is a bit weird. Maybe `modifyIptables` is a better choice of words? We should at least try to use something more descriptive of what it actually does.

Reply:
Once again, it is because of the KIAM flag

```
--iptables
```

Reply:
ok